features/page_objects/login_page.py

```diff
@@ -1,4 +1,6 @@
 LOGIN_BTN = '#login2'
+LOGIN_FORM_BTN = '#logInModal > div > div > div.modal-footer > button.btn.btn-primary'
+LOGOUT_BUTTON = '#logout2'
```

Comment:
minor: you should use the same notation for the elements you map

example, if you abbreviated `button` as `btn`, you should follow that naming convention